### PR TITLE
Feature/develop/task2

### PR DESCRIPTION
**Pull request by lsoulworkerl**
Create code on Python for get data.
Ino files for arduino slave/master.

### CODE REVIEW

File: lab2/master.ino [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+    } 	
+    
+  }
+  	delay(10); // Delay a little bit to improve simulation performance
+
+} // end loop
+
+
+// The function which sets hexadecimal bits based on the char from keypad
+void setBits(char keypressed)
```
Comment (andrii-s-shpitser-lpnu-ua) [inline lab2/master.ino:112]: what this code for?

File: lab2/slave.ino [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  	pinMode(CLK, INPUT);
+	pinMode(MOSI, INPUT);
+    pinMode(MISO, OUTPUT);
+	pinMode(SS, INPUT);
+  
+  	// set initial states of output pins
+  	digitalWrite(MISO, LOW);
+  
+  	// set up the LCD's number of columns and rows:
+  	lcd.begin(16, 2);
```
Comment (andrii-s-shpitser-lpnu-ua) [inline lab2/slave.ino:32]: Why you are using LCD?

**Review by andrii-s-shpitser-lpnu-ua — CHANGES_REQUESTED** [review]

File: lab2/master.ino (hunk 1 above)
Comment (lsoulworkerl) [reply lab2/master.ino:112]: passes the read data to a function that says which bits should be transmitted

File: lab2/slave.ino (hunk 2 above)
Comment (lsoulworkerl) [reply lab2/slave.ino:32]: for convenience

**Review by andrii-s-shpitser-lpnu-ua — APPROVED** [review]

**Pull request merged**